pkg/updates/updates.go

```diff
@@ -261,15 +260,11 @@ func (u *Updater) downloadAndInstall(rawUrl string) error {
 	}
 
 	g := new(getter.HttpGetter)
-	tempDir, err := ioutil.TempDir("", "lazygit")
-	if err != nil {
-		return err
-	}
-	defer os.RemoveAll(tempDir)
-	u.Log.Info("Temp directory is " + tempDir)
+	configDir := filepath.Dir(u.Config.GetUserConfigPath())
```

Comment:
I wonder if it's worth having u.Config.GetUserConfigDir() rather than `Path()` given we'll often end up calling `.Dir` on it

Reply:
Yep, you are right. Just thought about future, that it might be needed some day to get the full path of config file.

Reply:
Anyway I made it this way in latest commit.